File: dojo-bff/dojo-bff/src/main/java/com/sipios/dojo/realestatebff/service/model/Realty.java
package com.sipios.dojo.realestatebff.service.model;

import java.math.BigDecimal;

public record Realty(
        int id,
        BigDecimal amount,
        BigDecimal rent,
        String label,
        String image,
        DeedTypeEnum deedType,
        int scoring,
        String lastScoreUpdate
) {
}

File: dojo-front/dojo-front/src/shared/interface/realty.ts
export interface Realty {
  id: number;
  amount: number;
  rent: number;
  label: string;
  image: string;
  deedType: REALTY_TYPE;
  scoring: number;
  lastScoreUpdate: Date;
}

export enum REALTY_TYPE {
  APARTMENT = 'APARTMENT',
  PARTICIPATIVE = 'PARTICIPATIVE',
}
